Jenkinsfile

```diff
@@ -63,91 +69,146 @@ pipeline {
           // This build is a little unusual; it actually has _two_ artifacts, this one
           // and one for store-pg-assembly.  sbtbuild isn't really set up to override names
           // for two separate artifacts, so here we override the primary, and we'll override
-          // the secondary down below when it's passed to dockerize.
+          // the secondary down below when it's passed to docker.
           sbtbuild.setSrcJar("soql-server-pg/target/soql-server-pg-assembly.jar")
           env.STORE_PG_ARTIFACT = "store-pg/target/store-pg-assembly.jar"
           sbtbuild.build()
         }
       }
     }
-    stage('Dockerize SoQL Server PG') {
+    stage('Docker Build') {
       when {
-        not { expression { isPr } }
+        allOf {
+          not { expression { isPr } }
+          not { expression { skip } }
+        }
       }
-      steps {
-        script {
-          lastStage = env.STAGE_NAME
-          if (params.RELEASE_BUILD) {
-            env.REGISTRY_PUSH = (params.RELEASE_DRY_RUN) ? 'none' : 'all'
-            env.DOCKER_TAG = dockerize_server.docker_build_specify_tag_and_push(params.RELEASE_NAME, sbtbuild.getDockerPath(project_wd_server), sbtbuild.getDockerArtifact(project_wd_server), env.REGISTRY_PUSH)
-          } else {
-            env.REGISTRY_PUSH = 'internal'
-            env.DOCKER_TAG = dockerize_server.docker_build('STAGING', env.GIT_COMMIT, sbtbuild.getDockerPath(project_wd_server), sbtbuild.getDockerArtifact(project_wd_server), env.REGISTRY_PUSH)
+      stages {
+        stage('Docker Build SoQL Server PG') {
+          steps {
+            script {
+              lastStage = env.STAGE_NAME
+              if (params.RELEASE_BUILD || isHotfix) {
+                env.VERSION = (isHotfix) ? env.HOTFIX_NAME : params.RELEASE_NAME
+                env.DOCKER_TAG = dockerize_server.docker_build_specify_tag_and_push(env.VERSION, sbtbuild.getDockerPath(project_wd_server), sbtbuild.getDockerArtifact(project_wd_server), 'none')
```

Comment:
With the fourth parameter 'none', this [function]((https://github.com/socrata/pipelines/blob/main/src/com/socrata/Dockerize.groovy#L131) ) will perform a docker build, but not a registry push.  The [docker_build](https://github.com/socrata/pipelines/blob/main/src/com/socrata/Dockerize.groovy#L83) function is similar.